ManUp/ManUp.m

```diff
@@ -265,10 +282,31 @@ - (void)checkAppVersion {
     NSComparisonResult minVersionComparisonResult = [ManUp compareVersion:installedVersion toVersion:minVersion];
     NSComparisonResult currentVersionComparisonResult = [ManUp compareVersion:installedVersion toVersion:currentVersion];
     
-    if (self.alertController) {
+    if (self.alertController && !self.maintenanceModeShown) {
```

Comment:
Maintenance mode should be shown in favour of an update alert. That is an update alert should be removed snd replaced with a maintenance alert